Logger.php

```diff
@@ -307,7 +307,7 @@ public static function absToRelPath($pathToConvert)
     {
         $pathAbs = str_replace(['/', '\\'], '/', $pathToConvert);
         $documentRoot = str_replace(['/', '\\'], '/', $_SERVER['DOCUMENT_ROOT']);
-        return $_SERVER['SERVER_NAME'] . str_replace($documentRoot, '', $pathAbs);
+        return ($_SERVER['SERVER_NAME'] ?? '') . str_replace($documentRoot, '', $pathAbs);
```

Comment:
What about an alternative name when in context CLI? E.g. "CLI Context" or something? Any suggestions?

Reply:
`localhost` a possible approach?

Reply:
Localhost could still be within a browser context, something different then?

Reply:
`cli`? In analogy with https://www.php.net/manual/en/function.php-sapi-name.php ?

Reply:
Yeah, that would work.
Question tho, the `$_SERVER['SERVER_NAME']` doesn't exist for localhost AND when running in cli?
Then `cli` may be misleading in localhost environment. Could we have instead then a catch all name?

My bad, it's been over 4 years that I worked in PHP

Reply:
I've set' the default to `cli`. Feel free to approve, or change 👍.

If I understand your remark correct, I believe `localhost` does reflect correct. 

![image](https://github.com/user-attachments/assets/50d10160-9000-4058-8baf-5aec6c88dd2c)
![image](https://github.com/user-attachments/assets/31375aeb-1684-450c-baa6-05968f6547a8)

Reply:
@advename any succes yet in getting these changes merged?

Reply:
Thanks for nudging me @DaanDeSmedt ! I'll merge it ASAP!
And thanks for your contribution